clang-tools-extra/clang-doc/Representation.h

```diff
@@ -104,6 +104,9 @@ struct Reference {
 
   bool mergeable(const Reference &Other);
   void merge(Reference &&I);
+  bool operator<(const Reference &Other) const {
```

Comment:
How stable are usrs? Perhaps we should be sorting on another property?

Reply:
My concern here is that we probably want clang-doc to output to be ordered consistently. While sorting by USRs may make it deterministic, it’s also likely that they won’t be ordered similarly run to run. For instance, I’d expect public class fields to be sorted by name.

Reply:
the main reason I used USR was because it was unique to every declaration std sort wouldn't deal with equal elements, but it seems from the doc llvm:;sort already deterministically sort equal elements So I changed it to sort by name